typhos/ui/widgets/positioner.ui

```diff
@@ -319,17 +319,23 @@ Screen</string>
         <item>
          <widget class="PyDMLabel" name="user_readback">
           <property name="sizePolicy">
-           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
-            <width>0</width>
+            <width>201</width>
             <height>25</height>
            </size>
           </property>
+          <property name="maximumSize">
```

Comment:
With dynamic sizing I wonder if setting maximum sizes will ever come back to bite us.  Admittedly I don't remember enough of the dynamic sizing details to answer this myself...

Reply:
It turns out that these input sub-widgets don't resize at all anyway, even with the size policy set to "expanding". I don't have any idea why this is the case. Before I went to fixed mode, the widget wasn't even taking up all of the available space.

In addition:
For the row widget: we want the readback to expand instead of the setpoint
For the square widget: we don't want anything to resize, e.g. we don't want to see the whole middle section balloon out and we don't want the square to stretch up and down.